all-spark-notebook/Dockerfile

```diff
@@ -20,11 +20,11 @@ RUN apt-get update && \
 USER $NB_USER
 
 # R packages
-RUN conda config --add channels r && \
+RUN conda config --add channels rdonnellyr && \
```

Comment:
Is adding the rdonnellyr channel required to get 3.3? Can we trust builds coming from that channel?